pending_changelog/1699.txt

```diff
@@ -0,0 +1,2 @@
+`broadcast_node_announcement` has been moved to `PeerManager` from `ChannelManager`.
+`PeerManager::new`'s new `current_time` argument must be set to the current time, not a counter, for any existing nodes which upgrade.
```

Comment:
Why bother saying "not a counter"? The counter was previously an implementation detail.

Also, shouldn't all nodes need to set it to the current time regardless if upgrading?